Clean up docs and meta files (#1721)
GoogleCloudPlatform/microservices-demo: README.md

| [![Screenshot of store homepage](/docs/img/online-boutique-frontend-1.png)](/docs/img/online-boutique-frontend-1.png) | [![Screenshot of checkout screen](/docs/img/online-boutique-frontend-2.png)](/docs/img/online-boutique-frontend-2.png) |

## Interactive quickstart (GKE)

[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fmicroservices-demo&shellonly=true&cloudshell_image=gcr.io/ds-artifacts-cloudshell/deploystack_custom_image)

## Quickstart (GKE)

1. Ensure you have the following requirements:
   - [Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project).
   - Shell environment with `gcloud`, `git`, and `kubectl`.

2. Clone the repository.

   ```sh
   git clone https://github.com/GoogleCloudPlatform/microservices-demo
   cd microservices-demo/
   ```

3. Set the Google Cloud project and region and ensure the Google Kubernetes Engine API is enabled.

   ```sh
   gcloud config set project <PROJECT_ID>

>    gcloud config set compute/region us-central1

Reviewer: Praise: Yet another decision (`us-central1`) we're removing from this quickstart. I like.

   gcloud services enable container.googleapis.com
   ```

   Substitute `<PROJECT_ID>` with the ID of your Google Cloud project.

4. Create a GKE cluster and get the credentials for it.

   ```sh

>    gcloud container clusters create-auto online-boutique

Author: -especially- for a quickstart where they just want to get going without having to make any decisions (standard vs. autopilot? hmmmmm....)

>    gcloud container clusters create-auto online-boutique

Reviewer: **Issue:** This `gcloud container clusters create-auto online-boutique` command gave me an error:
```
ERROR: (gcloud.container.clusters.create-auto) ResponseError: code=400, message=Autopilot clusters must be regional clusters.
```
This might be specific to my `gcloud` configuration — but I can see our users encountering the same issue.
Let's specify the region:
```
gcloud container clusters create-auto online-boutique --region=us-central1
```

Author: Fixed!

>    gcloud container clusters get-credentials online-boutique

Reviewer: **Suggestion:**
We can remove this command:
```
gcloud container clusters get-credentials online-boutique
```

`gcloud container clusters create-auto` takes care of retrieving the credentials; albeit, this [isn't explicitly documented](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create-auto).
It even sets the created cluster to the current Kubernetes context.
See my [logs from my local run test](https://screenshot.googleplex.com/7T5egWjo5z3mmC7.png).

Author: Fixed!

>    ```

Reviewer: **Suggestion:** Let's add a note about the time `create-auto` takes.
Something like:
```
It can take about 4 minutes for the cluster to be created.
```
It took me 4 minutes.

Author: Fixed!

> 5. Deploy Bank of Anthos to the cluster.

Reviewer: ```suggestion
5. Deploy Online Boutique to the cluster.
```

Author: LOL!

   ```
   NAME                                     READY   STATUS    RESTARTS   AGE
   adservice-76bdd69666-ckc5j               1/1     Running   0          2m58s
   cartservice-66d497c6b7-dp5jr             1/1     Running   0          2m59s
   checkoutservice-666c784bd6-4jd22         1/1     Running   0          3m1s
   currencyservice-5d5d496984-4jmd7         1/1     Running   0          2m59s
   emailservice-667457d9d6-75jcq            1/1     Running   0          3m2s
   frontend-6b8d69b9fb-wjqdg                1/1     Running   0          3m1s
   loadgenerator-665b5cd444-gwqdq           1/1     Running   0          3m
   paymentservice-68596d6dd6-bf6bv          1/1     Running   0          3m
   productcatalogservice-557d474574-888kr   1/1     Running   0          3m
   recommendationservice-69c56b74d4-7z8r5   1/1     Running   0          3m1s
   redis-cart-5f59546cdd-5jnqf              1/1     Running   0          2m58s
   shippingservice-6ccc89f8fd-v686r         1/1     Running   0          2m58s
   ```

7. Access the web frontend in a browser using the frontend's external IP.

   ```
   kubectl get service frontend-external | awk '{print $4}'
   ```

>    Visit `https://EXTERNAL_IP` to access your instance of Bank of Anthos.

Author: ```suggestion
   Visit `https://EXTERNAL_IP` to access your instance of Online Boutique.
```

>    Visit `https://EXTERNAL_IP` to access your instance of Online Boutique.

Reviewer: ```suggestion
   Visit `http://EXTERNAL_IP` to access your instance of Online Boutique.
```

Author: Fixed!

8. Once you are done with it, delete the GKE cluster.

   ```

>    gcloud container clusters delete online-boutique

Reviewer: **Issue:**
I had a GKE standard cluster (`us-central1-a`) called  `online-boutique` in a different project, and this `delete` command first attempted to delete that cluster instead.

So let's specify the `--region`.
```suggestion
   gcloud container clusters delete online-boutique --region=us-central1
```

It's strange because I even re-ran `gcloud config set project MY_PROJECT_ID` before running the `delete` command `gcloud config set compute/region us-central1`.
See [my logs here](https://screenshot.googleplex.com/44pffbbNjNyc4cu.png).

Author: Fixed!

>    ```

Reviewer: **Thought:**
For some reason, even after Online Boutique was successfully deployed, my cluster was "repairing" (for at least 10 minutes, possibly longer — I didn't wait to find out).
See my [logs here](https://screenshot.googleplex.com/44pffbbNjNyc4cu.png). See my [UI here](https://screenshot.googleplex.com/92foLdfTzECRyNo.png).
```
ERROR: (gcloud.container.clusters.delete) Some requests did not succeed:
 - ResponseError: code=400, message=Operation operation-1682103148437-d628c3f9-139c-4048-a82a-8b4749943059 is currently repairing cluster online-boutique. Please wait and try again once it is done.
```

So I couldn't delete the cluster immediately.
If you're aware that this is a recurring and common issue, feel free to add a note.
Otherwise, no action required — might be a random error.

This is really something for the autopilot PMs.

Author: Added a quick note

| [![Screenshot of store homepage](/docs/img/online-boutique-frontend-1.png)](/docs/img/online-boutique-frontend-1.png) | [![Screenshot of checkout screen](/docs/img/online-boutique-frontend-2.png)](/docs/img/online-boutique-frontend-2.png) |

## Interactive quickstart (GKE)

[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fmicroservices-demo&shellonly=true&cloudshell_image=gcr.io/ds-artifacts-cloudshell/deploystack_custom_image)

## Quickstart (GKE)

1. Ensure you have the following requirements:
   - [Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project).
   - Shell environment with `gcloud`, `git`, and `kubectl`.

2. Clone the repository.

   ```sh
   git clone https://github.com/GoogleCloudPlatform/microservices-demo
   cd microservices-demo/
   ```

3. Set the Google Cloud project and region and ensure the Google Kubernetes Engine API is enabled.

   ```sh
   export PROJECT_ID=<PROJECT_ID>
   export REGION=us-central1
   gcloud services enable container.googleapis.com \

>      --project=${PROJECT_ID} --region=${REGION}

Reviewer: *Typo:*

```suggestion
     --project=${PROJECT_ID}
```

Error I got:
```
ERROR: (gcloud.services.enable) unrecognized arguments: --region=us-central1
```

Author: 🤦
